docker-compose.build.yaml

```diff
@@ -1,18 +1,18 @@
 version: "3.7"
 
 services:
-  db:
-    image: airbyte/db:${BUILD_TAG}
+  init:
```

Comment:
i just switched the order of init and db here to match how we do it in all other coppose files. the diff looks alot scarier than what actually happened.